packages/styles/scss/theme/_foundation.scss

```diff
@@ -26,6 +26,8 @@
   --ilo-fonts-monospace: monospace;
   --ilo-line-height: 1.46;
 
+  --ilo-types-weight-label: 700;
```

Comment:
@GGKapanadze Shouldn't this be --ilo-font-weight-bold? We are moving away from context based naming right?

Reply:
This should be declared as a component variable